src/main/java/com/oath/halodb/FileUtils.java

```diff
@@ -30,6 +32,17 @@ static void createDirectoryIfNotExists(File directory) throws IOException {
         }
     }
 
+    static void deleteDirectory(File dir) throws IOException {
```

Comment:
```org.apache.commons.io.FileUtils``` is doing a better job

Reply:
As there's already a class with same name. I don't want to introduce apache FileUtils as it would be somewhat confused.

Reply:
Then add it to FileUtils and call FileUtils here

Reply:
It's quite a simple function. We could just refer the implementation of the lib.

Reply:
Just noticed, it is already in FileUtils. The name ```deleteDirectory``` is not appropriate here because they function with assumption no embedded folder in it. Because it is only used to delete snapshot. How about rename to ```deleteSnapshotDir``` ?